Modules/CustomRolesHelper.cs

```diff
@@ -1689,6 +1690,21 @@ public static bool CheckAddonConfilct(CustomRoles role, PlayerControl pc)
                 if ((pc.GetCustomRole().IsCrewmate() && !Silent.CanBeOnCrew.GetBool()) || (pc.GetCustomRole().IsNeutral() && !Silent.CanBeOnNeutral.GetBool()) || (pc.GetCustomRole().IsImpostor() && !Silent.CanBeOnImp.GetBool()))
                     return false;
                 break;
+            case CustomRoles.Susceptible:
+                if ((pc.GetCustomRole().IsCrewmate() && !Susceptible.CanBeOnCrew.GetBool()) || (pc.GetCustomRole().IsNeutral() && !Susceptible.CanBeOnNeutral.GetBool()) || (pc.GetCustomRole().IsImpostor() && !Susceptible.CanBeOnImp.GetBool()))
+                return false;
+                break;
```

Comment:
Susceptible was removed, so this code is not needed
![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/104814436/b54e2f1a-57d2-491d-906f-72c0524f722f)

Reply:
That is sunglasses not susceptible 😅

Reply:
uh... this is a image of the sunglasses add-on? I added susceptible case in this pr, because I forgot in the other 😅